qiskit/compiler/assemble.py

```diff
@@ -77,24 +77,24 @@ def assemble(experiments,
             Random seed to control sampling, for when backend is a simulator
 
         qubit_lo_freq (list):
-            List of default qubit LO frequencies. Will be overridden by
+            List of default qubit lo frequencies in Hz. Will be overridden by
             `schedule_los` if set.
 
         meas_lo_freq (list):
-            List of default meas LO frequencies. Will be overridden by
-            `schedule_los` if set.
+            List of default measurement lo frequencies in Hz. Will be overridden
+            by `schedule_los` if set.
 
         qubit_lo_range (list):
-            List of drive LO ranges used to validate that the supplied qubit LOs
-            are valid.
+            List of drive lo ranges each of form `[range_min, range_max]` in Hz.
+            Used to validate that the supplied qubit los are valid.
```

Comment:
Has this been merged or rebased? I thought I had removed the redundancy in these lines a while ago
```suggestion
            Used to validate the supplied qubit frequencies.
```